environment.yml

```diff
@@ -17,7 +17,9 @@ dependencies:
   - pytest-mock
   - requests-mock
   - httpx
+  # Runtime dependencies
   - fire
+  - dacite
   - tbump
```

Comment:
Why is `tbump` a runtime dependency?

Reply:
Good point, I synced the dependencies, also with `setup.cfg`.